docs/index_es.html

```diff
@@ -0,0 +1,118 @@
+    <meta property="og:image:height" content="675">
+    <meta property="og:image" content="https://1x.engineer/img/1x.engineer.png">
+
+    <meta name="twitter:title" content="1x Engineer">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@bitandbang">
+    <meta name="twitter:description" content="El sitio web oficial de los y las ingenieras 1x en el mundo.">
+    <meta name="twitter:image" content="https://1x.engineer/img/1x.engineer.png">
+
+  </head>
+  <body>
+    <main>
+    <section>
+      <h1>Qué es un/a ingeniero/a 1x?</h1>
+      <p>Es posible que ya haya oído hablar de un ingeniero 10x. Probablemente con demasiada frecuencia, en realidad. Si existe un ingeniero 10x,
+        seguramente también debe haber un/a ingeniero/a 1x.</p>
+      <p>¡Por supuesto que lo/la hay! Profundicemos en una lista no exhaustiva de las cualidades que componen un/a ingeniero/a 1x.</p>
+    </section>
+    <section class="nes-container with-title">
+      <h2 class="title">Un/a Ingeniero/a 1x</h2>
+      <ul class="lists">
+          <li>Busca en Google, Duckduckgo, Bing o donde quiera cuando no está seguro/a de lo que pasa.</li>
+          <li>Copia/pega fragmentos de código de Stack Overflow, Glitch, Codepen o donde sea que encuentren respuestas.</li>
+          <li>Da crédito donde se debe crédito.</li>
+          <li>Creates community and shares knowledge.</li>
+          <li>Dedica tiempo a cosas fuera de la ingeniería, como pasatiempos, amigos y familia.</li>
+          <li>Tiene un horario que les permite mantener un equilibrio saludable entre el trabajo y la vida y también respeta los límites de tiempo de los demás.</li>
+          <li>No se mide por puntajes de contribución arbitrarios en ningún sitio web, y tampoco juzga a otras personas por los suyos.</li>
+          <li>Escribe código que <span title="No se preocupe por enviar un PR para esto, es un error typo intencional, ya sabe, ¡porque creamos errores!">&emdash;</span> <em>gasp</em> &emdash; tiene bugs.</li>
+          <li>Escribe código que otro/as pueden leer.</li>
+          <li>Lee los documentos.</li>
+          <li>Actualiza los documentos.</li>
+          <li>No es necesario que le apasione el código que escribe o los problemas que resuelve, pero puede ser que si.</li>
+          <li>No se sorprende cuando alguien no sabe algo.</li>
+          <li>Está dispuesto/a y es capaz de colaborar con otros.</li>
+          <li>Celebra públicamente a otro/as por sus victorias.</li>
+          <li>Hace preguntas antes de proporcionar comentarios críticos.</li>
+          <li>Da retroalimentación dura en privado.</li>
+          <li>Trata a otras personas como a <em>el/ella</em> le gustaría ser tratado/a.</li>
+          <li>Proporciona revisiones de código y retroalimentación a sus compañero/as que son constructivos, útiles y presentados con tacto, ayudando a sus compañero/as a crecer personal y profesionalmente.</li>
```

Comment:
I think this should be feminine, since both "revisiones" and "retroalimentación" are feminine.

```suggestion
          <li>Proporciona revisiones de código y retroalimentación a sus compañero/as que son constructivas, útiles y presentadas con tacto, ayudando a sus compañero/as a crecer personal y profesionalmente.</li>
```